actioncable/lib/action_cable/subscription_adapter/inline.rb

```diff
@@ -26,7 +28,11 @@ def shutdown
 
       private
         def subscriber_map
-          @subscriber_map || @server.mutex.synchronize { @subscriber_map ||= new_subscriber_map }
+          @subscriber_map || @server.mutex.synchronize do
+            @subscriber_map ||= new_subscriber_map.tap do |subscriber_map|
+              subscriber_map.compress_with(@compressor)
+            end
+          end
```

Comment:
What's the benefit here?

Reply:
Prefer inline?

`compress_with` was originally intended to give adapters flexibility to change these defaults.

If all adapters will have the same values it is probably better to move the compressor initialization to the `ActionCable::SubscriptionAdapter::Base` constructor as suggested by @skipkayhil. I only see the problem of compatibility with external adapters when doing that.